assets/js/modules/analytics/components/common/GA4ActivateSwitch.js

```diff
@@ -67,13 +71,22 @@ export default function GA4ActivateSwitch( props ) {
 			<Switch
 				label={
 					<Fragment>
-						{ __(
-							'Activate Google Analytics 4 and place code on your site.',
-							'google-site-kit'
-						) }{ ' ' }
-						<Link href={ documentationURL } external>
-							{ __( 'Learn more', 'google-site-kit' ) }
-						</Link>
+						{ createInterpolateElement(
+							__(
+								'Activate Google Analytics 4 and place code on your site. <LearnMoreLink />',
```

Comment:
The same here, maybe better to use the link label in the main translatable string:
```suggestion
								'Activate Google Analytics 4 and place code on your site. <a>Learn more</a>',
```